recipes-kernel/linux/linux-raspberrypi_4.4.bbappend

```diff
@@ -0,0 +1,22 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI += "file://support_rpi_pitft35.patch \
```

Comment:
I think, support_rpi_pitft35.patch should be support_rpi_pitft35_44.patch.
Is this what you intended?